demos/coreHTTP/http_demo_s3_download_multithreaded.c

```diff
@@ -30,17 +30,19 @@
  * This example, using a pre-signed URL, resolves a S3 domain, establishes a TCP
  * connection, validates the server's certificate using the root CA certificate
  * defined in the config header, then finally performs a TLS handshake with the
- * HTTP server so that all communication is encrypted.
+ * HTTP server so that all communication is encrypted.The AWS S3 service validates
```

Comment:
The language on lines 30-33 is a bit awkward – server certificate validation is part of the TLS handshake, not a separate operation.

Reply:
Good catch, will fix it